app/components/account/browse/account-list/account-list.component.ts

```diff
@@ -75,7 +75,11 @@ export class AccountListComponent extends ListBaseComponent implements OnDestroy
         if (this.isAccountFavorite(accountId)) {
             this.accountService.unFavoriteAccount(accountId);
         } else {
-            this.accountService.favoriteAccount(accountId);
+            this.accountService.favoriteAccount(accountId).subscribe({
+                complete: () => {
```

Comment:
you can just do subscribe(() => {

});

here instead as it should emit multiple values

Reply:
cool, thanks.

Reply:
just changed it and now the favorite reverted back to its old behavior ...

Reply:
changed to this: 

            this.accountService.favoriteAccount(accountId).subscribe(() => {
                this.changeDetector.markForCheck();
            });

and its not being called any more. should i use next: () => {} instead?